content/secrets/10-walkthrough-step-07/copilot.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+Again - copilot is on autopilot.   The `init` command will deploy the application to a test environment when you choose the option to deploy a local environment.   If you choose not to deploy to a local environment, the cli will output the following commands
+
+```bash
+- Run `copilot env init --name test --profile default --app ecsworkshop` to create your staging environment.
+- Update your manifest copilot/ecs-secrets-service/manifest.yml to change the defaults.
+- Run `copilot svc deploy --name ecs-secrets-service --env test` to deploy your service to a test environment.
+```
+
+The last step for this tutorial is to copy the LoadBalancer URL from the last output and run:
+
+```bash
+curl ECSST-Farga-xxxxxxxxxx.yyyyy.elb.amazonawss.com/migrate | jq
+```
+
+This creates the database schema for the sample application.  To view the app, open a browser and go to the Loadbalancer URL `ECSST-Farga-xxxxxxxxxx.yyyyy.elb.amazonaws.com`:
+![Secrets Todo](/images/secrets-todo.png)
+
+This is a fully functional todo app.  Try creating, editing, and deleting todos.  Using the information output from deploy along with the secrets stored in Secrets Manager, connect to the Postgres Database using a database client or the `psql` command line tool to browse the database. 
```

Comment:
I deployed the app and can't create TODO items FYI. No errors in the console in browser, and here are logs from svc:

```
AWSReservedSSO_AdministratorAccess_7dffd66186439611:~/environment/secretecs (main) $ copilot svc logs --follow
Found only one deployed service todo-app in environment test
copilot/todo-app/aef2c561   schema: undefined,
copilot/todo-app/aef2c561   table: undefined,
copilot/todo-app/aef2c561   column: undefined,
copilot/todo-app/aef2c561   dataType: undefined,
copilot/todo-app/aef2c561   constraint: undefined,
copilot/todo-app/aef2c561   file: 'parse_relation.c',
copilot/todo-app/aef2c561   line: '1159',
copilot/todo-app/aef2c561   routine: 'parserOpenTable'
copilot/todo-app/aef2c561 }
copilot/todo-app/aef2c561 undefined
copilot/todo-app/aef2c561   where: undefined,
copilot/todo-app/aef2c561   schema: undefined,
copilot/todo-app/aef2c561   table: undefined,
copilot/todo-app/aef2c561   column: undefined,
copilot/todo-app/aef2c561   dataType: undefined,
copilot/todo-app/aef2c561   constraint: undefined,
copilot/todo-app/aef2c561   file: 'parse_relation.c',
copilot/todo-app/aef2c561   line: '1159',
copilot/todo-app/aef2c561   routine: 'parserOpenTable'
copilot/todo-app/aef2c561 }
```